proto/debugpb.proto

```diff
@@ -259,3 +263,10 @@ message GetAllRegionsInStoreRequest {
 message GetAllRegionsInStoreResponse {
     repeated uint64 regions = 1;
 }
+
+message ResetToVersionRequest {
+    uint64 ts = 1;
+}
+
+message ResetToVersionResponse {
+}
```

Comment:
Pleaase add an empty line here.

Reply:
> Do we need a resp head here?

In fact, there're no infomation we can return here.